packages/flame/lib/src/sprite_animation_ticker.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:async';
 
 import 'package:flame/components.dart';
+import 'package:flutter/material.dart';
```

Comment:
```suggestion
import 'package:meta/meta.dart';
```
Import meta here instead of material